packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewer/__tests__/PdfViewer.test.tsx

```diff
@@ -7,6 +7,7 @@ import { document as doc } from 'components/DocumentPreview/__fixtures__/Art Eff
 
 describe('PdfViewer', () => {
   it('renders PDF document', async () => {
+    // @ts-ignore pdfWorkerUrl is required, but needs to be undefined for tests
```

Comment:
This feels weird. If we have to twist our component in a way it shouldn't be used in order to test it, then that sounds like a bad test case. Why can't you pass in a blank or dummy string?

Reply:
The unit tests has always used the pdf.js `GlobalWorkerOptions.workerSrc` prop rather than the `GlobalWorkerOptions.workerPort` prop. This used to be defined in the production code. In this PR, I moved that out to the `setupTests` file so that all tests that use PdfViewer will have a worker set up already. Adding a `pdfWorkerUrl` prop to this component would set up the `GlobalWorkerOptions.workerPort` prop, which could break the worker (although I haven't actually tried it, so I'm not sure of that).

I can definitely add a more substantial comment here (and point to `setupTests`) if that would make the code clearer.